src/components/Auth/Forms/login.tsx

```diff
@@ -130,14 +130,14 @@ const LoginForm = () => {
         <div className="m-1 text-center">
           <div>
             <Link href="/auth/forgot-password">
-              <span className="text-xs text-gray-400 underline">
+              <span className="text-[13px] text-gray-400 underline">
```

Comment:
We recommend not using arbitrary values but if these are due to spacing issues/line-height issues, check the leading classes

Reply:
That's more for increasing the font size, but we've now got it sorted :)